source/components/parser/psobject.c

```diff
@@ -768,7 +768,8 @@ AcpiPsCompleteFinalOp (
     ACPI_PARSE_OBJECT       *Op,
     ACPI_STATUS             Status)
 {
-    ACPI_STATUS             Status2;
+    ACPI_STATUS             ReturnStatus = AE_OK;
```

Comment:
We should change this to ReturnStatus = Status. This will preserve the old behavior if Op is NULL.

Reply:
That is a good point. I will change the code and commit message. Thank you for your advice.